.github/workflows/sync-design-tokens.yml

```
@@ -11,12 +11,20 @@ jobs:
       - uses: actions/checkout@v2.3.5
         env:
           LANG: en_US.UTF-8
+      - uses: pnpm/action-setup@v2.2.1
+        with:
+          version: 6.16.0
+      - name: Use Node.js
+        uses: actions/setup-node@v3.0.0
+        with:
+          node-version: 17
+          cache: 'pnpm'
       - name: Generate json files
         env:
           URL: https://raw.githubusercontent.com/scaleway/design-tokens/main/tokens.json
         run: |
           /scripts/figma-synchronise/tokens.sh $URL
-          yarn run format
+          pnpm format
```

How could that have worked without node 🤔 

```suggestion
          pnpm run  format
```

---

The base `ubuntu` image from github actions might have nodejs and yarn pre-installed